ogre2/src/Ogre2Material.cc

```diff
@@ -1205,7 +1205,7 @@ void Ogre2Material::SetTextureMapDataImpl(const std::string& _name,
   {
     auto data = _img->RGBAData();
 
-    texture->setPixelFormat(Ogre::PFG_RGBA8_UNORM_SRGB);
+    texture->setPixelFormat(Ogre::PFG_RGBA8_UNORM);
```

Comment:
can you give this a try:

```
    Ogre::PixelFormatGpu format = Ogre::PFG_RGBA8_UNORM;
    if (this->ogreDatablock->suggestUsingSRGB(_type))
      format = Ogre::PFG_RGBA8_UNORM_SRGB;
    texture->setPixelFormat(format);
```

Reply:
this will likely set diffuse map to be sRGB while keeping metalness and roughness textures to be non-sRGB

Reply:
@ahcorde checking if you still see inconsistent visual results with these changes?